CRM/Contact/Form/Task/SaveSearch.php

```diff
@@ -242,4 +240,17 @@ public function postProcess() {
     }
   }
 
+  /**
+   * Set form defaults.
+   *
+   * return array
+   */
+  public function setDefaultValues() {
+    $defaults = array();
+    if (empty($defaults['parents'])) {
+      $defaults['parents'] = CRM_Core_BAO_Domain::getGroupId();
```

Comment:
@eileenmcnaughton I notice here that you are setting the default as having a parent group here, I am just wondering how that works with multisite given iirc in multisite extension we are looking at default being the domain org contact not the domain group